profiles/common/modules/features/nexteuropa_trackedchanges/nexteuropa_trackedchanges.admin.inc

```diff
@@ -0,0 +1,700 @@
+  }
+
+  $form['nexteuropa_trackedchanges_rebuild_elapse_time'] = array(
+    '#title' => t('Tracked changes table refresh frequency'),
+    '#type' => 'textfield',
+    '#field_suffix' => t('seconds'),
+    '#size' => 3,
+    '#description' => t('Set the elapse time between 2 refreshes of listing 
+    entities having tracked changes. 0 second means the list will be refreshed
+    each time the drupal cron is executed.'),
+    '#default_value' => variable_get('nexteuropa_trackedchanges_rebuild_elapse_time'),
+    '#required' => TRUE,
+  );
+  $form['save'] = array(
+    '#type' => 'submit',
+    '#value' => t('Save configuration'),
+    '#submit' => array('nnexteuropa_trackedchanges_rebuild_elapse_time_submit'),
+    '#validate' => array('nexteuropa_trackedchanges_rebuild_elapse_time_validate'),
+  );
+
+  $form['table_status'] = array(
+    '#markup' => $output,
+  );
+  if (!$is_lock) {
+    $form['force_rebuild'] = array(
+      '#type' => 'submit',
+      '#value' => t('Force scanning'),
+      '#submit' => array('nexteuropa_trackedchanges_table_rebuilt_submit'),
+      '#description' => t('Force the system to scan entities immediately. This process can take times.'),
+    );
+  }
+  else {
+    $form['table_status'] = array(
+      '#markup' => t('The rebuild of the logs table is under way.'),
+    );
+  }
+
+  return $form;
+}
+
+/**
+ * Form submit callback.
+ *
+ * @see nexteuropa_trackedchanges_rebuild_status()
+ */
+function nnexteuropa_trackedchanges_rebuild_elapse_time_submit($form, &$form_state) {
+  $values = $form_state['values'];
+  $elapse_time_value = (!empty($values['nexteuropa_trackedchanges_rebuild_elapse_time'])) ? $values['nexteuropa_trackedchanges_rebuild_elapse_time'] : 0;
+
+  variable_set('nexteuropa_trackedchanges_rebuild_elapse_time', $elapse_time_value);
+}
+
+/**
+ * Form validate callback.
+ *
+ * @see nexteuropa_trackedchanges_rebuild_status()
+ */
+function nexteuropa_trackedchanges_rebuild_elapse_time_validate($form, &$form_state) {
+  $values = $form_state['values'];
+  $elapse_time_value = $values['nexteuropa_trackedchanges_rebuild_elapse_time'];
+  if (!is_numeric($elapse_time_value) || ($elapse_time_value < 0)) {
+    form_set_error('nexteuropa_trackedchanges_rebuild_elapse_time',
+      t('The "refresh frequency" field must be a positive number.'));
+  }
+}
+
+/**
+ * Submit form call back triggering the netc_info table.
+ *
+ * @see nexteuropa_trackedchanges_rebuild_status()
+ */
+function nexteuropa_trackedchanges_table_rebuilt_submit($form, &$form_state) {
+
+  // Empty the table and "lock" it.
+  variable_set('nexteuropa_trackedchanges_rebuild_lock', 1);
+  // It is a rebuild, then we flush the tables.
+  _nexteuropa_trackedchanges_clean_db();
+
+  $operations = array();
+  // The 2nd argument is useless in this case because the default Batch API
+  // process fulfils function requirements.
+  $operations[] = array('_nexteuropa_trackedchanges_revisions_scanning_process', array(''));
```

Comment:
It is not the initialization, it is the actual value used by the batch. I experienced `array()` and I got weird behaviours.